src/styles/colorManipulator.test.js

```diff
@@ -187,6 +187,17 @@ describe('utils/colorManipulator', () => {
       assert.strictEqual(fade('hsla(0, 100%, 50%, 0.2)', 0.5), 'hsla(0, 100%, 50%, 0.5)');
     });
 
+    it('sets the transparency of a color to the transparency of the color provided', () => {
+      assert.strictEqual(
+        fade('rgba(255, 0, 0, 0.2)', 'rgba(1, 2, 3, 0.5)'),
```

Comment:
Do we want to support this API? This looks counter-intuitive to me.

Reply:
My first reaction was "yuck", so I"m glad it wasn't just me! 😄

Reply:
Do you think the approach of reusing the existing theme variable in this way is basically correct? Then I would move the logic for extracting and applying transparency to its own function.

Reply:
@oliviertassinari In my opinion, adding a new variable causes the following problem:

The transparency of hover effects is configured in two variables and must be kept consistent with each other. If a user changes only one of the two variables, the hover effects of the components are inconsistent. The user does not necessarily notice this error directly; and later the user wonders why the components of his user interface are inconsistent with each other.

To avoid this error, I would suggest to use only one variable and extract the required information (the degree of transparency) via a help function. This simplifies the configuration of the theme for the user.

Reply:
> The transparency of hover effects is configured in two variables and must be kept consistent with each other.

@SebastianSchmidt I have realized that the current fade value (0.04) is close to imperceptible with the dark theme. It's a tradeoff, I can live with it. But why not addressing the duplication issue 👍 . The helper function can do it. Should we keep the intermediary result (`hoverOpacity`) in the theme or compute it everytime?